src/auditing.md

```diff
@@ -377,6 +378,78 @@ new-context=?: exe="/usr/bin/newrole" hostname=? addr=?
 terminal=/dev/pts/0 res=failed'
 ```
 
+## Capability Audit Exemptions
+
+In the general case a rejected capability check will result in an audit event.
+There are however some instances in the kernel where denied capability checks
+are not audited, which could lead to differences in behavior between enforcing
+and permissive mode.
+
+List of exemptions (no guarantee for comleteness)(Locations are based on kernel v6.4 unless otherwise specified):
+
+- *fs/proc/base.c#L1110*,
+  *fs/proc/base.c#L1129*
+
+  If not granted *CAP_SYS_RESOURCE* the OOM kill score adjustment min value is not set.
+
+- *fs/overlayfs/inode.c#L422*,
+  *fs/xattr.c#L1298*
+
+  If not granted *CAP_SYS_ADMIN* in its namespace extended attributes in the *trusted* namespace are not listed.
+
+- *fs/xfs/xfs_fsmap.c#L868*
+
+  If not granted *CAP_SYS_ADMIN* the XFS data device's bnobt is queried instead of rmapbt.
+
+- *fs/xfs/xfs_ioctl.c#L1199*,
+  *fs/xfs/xfs_iops.c#L709*
+
+  If not granted *CAP_FOWNER* XFS quota checks on transactions are performed.
+
+- *kernel/capability.c#L518*
+
+  If not granted *CAP_SYS_PTRACE* several tracing related operations are not permitted.
```

Comment:
Wouldn't it make more sense to enumerate the callers of `ptracer_capable()` in this list?  This seems like more of a utility function than the sort of call-site to enumerate here.  Grepping ptracer_capable looks like it only has 3 callers, so it wouldn't bloat the list too much to enumerate them instead of listing this call site.